src/source/ObjcGenerator.scala

```diff
@@ -178,7 +178,7 @@ class ObjcGenerator(spec: Spec) extends Generator(spec) {
     val self = marshal.typename(objcName, r)
 
     refs.header.add("#import <Foundation/Foundation.h>")
-    refs.body.add("!#import " + q(spec.objcIncludePrefix + (if (r.ext.objc) "../" else "") + marshal.headerName(ident)))
+    refs.body.add("!#import " + q(spec.objcIncludePrefix + (if (r.ext.objc) spec.objcExtendedRecordIncludePrefix else "") + marshal.headerName(ident)))
```

Comment:
I am sort of confused now.  Originally, I set off to do this because the special ../'s that I saw in the code were ending up in my extended header includes.  I didn't, and still don't, fully understand the full scope of what is going on with all this, but in looking at your comment and reading the code further, it seems that objcIncludePrefix could have been all I needed, except for the issue with the ../ and now that I see a comment in issue 193 about it.  I found the change, though it isn't that new, in commit b38789e6f8b0.

So, I think what I may be asking here, is does the change I am trying to make even need to be there?  Should instead, the hard ../ just be removed or changed?